CONFIGURATION_GUIDE.md

```diff
@@ -63,20 +63,28 @@ memory. Empirically, RSS worker off-heap memory should be set to `(numDirs  * bu
+|               rss.master.port               |                                                   |                                                                                                                                                                                                Port of RSS Master                                                                                                                                                                                                 |
+|         rss.worker.graceful.shutdown        |                     false                         |                                                       When true, during worker shutdown, the worker will wait for all released slots to be committed or destroyed in time of `rss.worker.checkSlots.timeout` and wait sorting partition files in time of `rss.worker.partitionSorterCloseAwaitTimeMs`.                                                                                                            |
+|         rss.worker.shutdown.timeout         |                      600s                         |                                                                                                                                                                                      The worker's graceful shutdown time.                                                                                                                                                                                         |
+|           rss.worker.recoverPath            | `${System.getProperty("java.io.tmpdir")}/recover` |                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| rss.worker.partitionSorterCloseAwaitTimeMs  |                     false                         |                                                                                                                                                            The wait time of waiting for sorting partition files during worker graceful shutdown.                                                                                                                                                                  |
```

Comment:
default to 120s, and the config name should be rss.worker.partitionSorterCloseAwaitTime

Reply:
> rss.worker.partitionSorterCloseAwaitTime

Done